concreep-refilled/control.lua

```diff
@@ -121,7 +115,7 @@ function _reinit_reentrant()
 	if global.reinit_level_running == 0 then
 		-- no loops running, check if we should start outer loop
 		global.no_work_counter = global.no_work_counter + 1
-		if global.no_work_counter >= NO_WORK_UPDATES_BEFORE_REINIT then
+		if global.no_work_counter >= DEFAULT_NO_WORK_UPDATES_BEFORE_REINIT then

```

Comment:
should get value from settings.global here

Reply:
done - sort of? I implemented them as hidden settings int the settings .lua as I thought we might make it user accessible at one point.